### PR TITLE
DOCSP-33172 added meta description for top 250 pages in docs-mongodb-shell

### PR DESCRIPTION
**Pull request by sarahemlin**
Hey @mdb-ashley!  I'm working on a ticket to add meta descriptions to the top 250 pages that don't have them.  Here are the ones for the mongosh repo.  In most cases, I copied and pasted a summarizing sentence from the page text, but please feel free to edit as you see fit.

Thanks!
Sarah

## STAGING
https://preview-mongodbsarahemlin.gatsbyjs.io/mongodb-shell/master/

## JIRA
https://jira.mongodb.org/browse/DOCSP-33172

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=655bdb505b686d6bd943dcf6

### CODE REVIEW

File: source/connect.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ Connect to a Deployment
    :values: tutorial
 
 .. meta:: 
+   :description: How to use the MongoDB Shell to connect to a MongoDB Atlas cloud-hosted deployment, a local deployment, or another remote host with MongoDB Shell.
```
Comment (mdb-ashley) [inline source/connect.txt:14]:
```suggestion
   :description: How to use the MongoDB Shell to connect to a MongoDB Atlas cloud-hosted deployment, a local deployment, or another remote host.
```

File: source/crud.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@ Perform CRUD Operations
    :values: reference
 
 .. meta:: 
+   :description: CRUD operations create, read, update, and delete documents within a MongoDB collection.
```
Comment (mdb-ashley) [inline source/crud.txt:14]:
```suggestion
   :description: Use the MongoDB Shell to create, read, update, and delete documents in a MongoDB collection.
```

File: source/crud/update.txt [hunk 3]
```diff
@@ -11,6 +11,7 @@ Update Documents
    :values: reference
 
 .. meta:: 
+   :description: MongoDB provides methods to update a single document, update multiple documents, and replace a document.
```
Comment (mdb-ashley) [inline source/crud/update.txt:14]:
```suggestion
   :description: Use the MongoDB Shell to update a single document, update multiple documents, or replace a document. 
```

File: source/index.txt [hunk 4]
```diff
@@ -6,6 +6,9 @@
    :name: genre
    :values: reference
 
+.. meta:: 
+   :description: Use the MongoDB Shell REPL environment to test queries and interact with the data in your MongoDB database.
```
Comment (mdb-ashley) [inline source/index.txt:10]:
```suggestion
   :description: Use the MongoDB Shell to test queries and interact with the data in your MongoDB database.
```

File: source/install.txt [hunk 5]
```diff
@@ -11,6 +11,7 @@ Install ``mongosh``
    :values: tutorial
 
 .. meta:: 
+   :description: Prerequisites, compatibility issues, and how to install mongosh, the MongoDB shell.
```
Comment (mdb-ashley) [inline source/install.txt:14]:
```suggestion
   :description: Prerequisites, compatibility issues, and installation instructions for mongosh, the MongoDB Shell.
```

File: source/run-commands.txt [hunk 6]
```diff
@@ -11,6 +11,7 @@ Run Commands
    :values: reference
 
 .. meta:: 
+   :description: Use run commands in mongosh to  format input or output, create or switch databases, terminate a query, and clear the console.
```
Comment (mdb-ashley) [inline source/run-commands.txt:14]:
```suggestion
   :description: Use run commands in the MongoDB Shell to create or switch databases, terminate a query, or clear the console.
```

**Review by mdb-ashley — CHANGES_REQUESTED** [review]
Thanks, Sarah! This looks great. I left some suggestions for your consideration.

**Comment by sarahemlin** [comment]
LGTM!  Thanks, @mdb-ashley!

**Pull request merged**